src/GeositeFramework/js/Export.js

```diff
@@ -105,6 +129,20 @@ require(['use!Geosite'],
                                 .wrapAll("<div class='print-legend-coll'></div>");
                         });
 
+                    function afterPrintHandler() {
+                        _.delay(previewDeferred.resolve, 250);
+                    }
+
+                    $.when(alertDeferred).then(function () {
+                        if (!!window.onafterprint) {//ie, ff, safari?
+                            window.onafterprint = afterPrintHandler;
+                        }
+                        window.print();
```

Comment:
It may be worth checking that your IE11 is up to date, when I tested against it with a base install there were some weird jQuery bugs that made the print feature unusable. Installing some security patch cleared it up.

A quick way to check, at least through my experience, is to open the IE11 developer tools and see if anything comes up. Before the patch there was no content in any of the panes.